### PR TITLE
Submit a container for Patchwork

### PR DESCRIPTION
**Pull request by ClaraKoehne**
# Docker Container for Patchwork
 
Patchwork is an alignment-based exon retrieval and concatenation program with phylogenomic applications. At the moment of writing, it is still under construction. The current release is a prototype. 
The Dockerfile we submit here can be used to build a Docker Image for Patchwork using `docker build` that can then be run in a container. 
 
 ## Checklist
 
 1. Misc
 - [x] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [x] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [x] LABEL base_image  
 - [x] LABEL version
 - [x] LABEL software.version  
 - [x] LABEL about.summary  
 - [x] LABEL about.home  
 - [x] LABEL about.license   
 - [x] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [x] LABEL about.documentation   
  - [x] LABEL about.license_file
  - [x] LABEL about.tags

### CODE REVIEW

File: patchwork/0.1.2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+WORKDIR /home/patchwork
+USER patchwork
+ENV VERSION 0.1.2-pre-alpha
+
+COPY --from=build /home/patchwork/patchwork-$VERSION patchwork-$VERSION
+COPY --from=build /home/patchwork/diamond diamond
+RUN ln -s patchwork-$VERSION/bin/patchwork patchwork
+ENV PATH /home/patchwork:$PATH
+
+CMD patchwork --help
```
Comment (osallou) [inline patchwork/0.1.2/Dockerfile]: please remove (or comment CMD)

**Review by osallou — CHANGES_REQUESTED** [review]
sofware name must be lowercase

**Comment by ClaraKoehne** [comment]
> Found a biotools entry matching the software name (https://bio.tools/patchwork), if this is the same software, please add the extra.identifiers.biotools label to your Dockerfile

It's not the same software. Do I still have to do something?

**Comment by osallou** [comment]
> > Found a biotools entry matching the software name (https://bio.tools/patchwork), if this is the same software, please add the extra.identifiers.biotools label to your Dockerfile
> 
> It's not the same software. Do I still have to do something?

nope, but not a good idea to reuse names...

File: patchwork/0.1.2/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+LABEL about.summary="Alignment-based Exon Retrieval and Concatenation with Phylogenomic Applications"
+LABEL about.home="https://github.com/fethalen/Patchwork"
+LABEL about.documentation="https://github.com/fethalen/Patchwork#readme"
+LABEL about.license="SPDX:GPL-3.0"
+LABEL about.license_file="https://github.com/fethalen/Patchwork/blob/main/LICENSE"
+LABEL about.tags="phylogenomics"
+
+# MAINTAINER ##################################################################################################################
+
+MAINTAINER Felix Thalén <felix.thalen@uni-goettingen.de>
```
Comment (osallou) [inline patchwork/0.1.2/Dockerfile]: getting some unicode decoding issues (certaintly here with special chars in name). Please do not use accents, special chars, ...

**Comment by osallou** [comment]
@ClaraKoehne merged and built, thanks

**Pull request merged**